index.html

```diff
@@ -2120,588 +2138,378 @@ <h3>Thing Root Object</h3>
         }
     },
     ...
+}
 </pre>
-</aside>
-
-</section> <!-- end of id="property-serialization-json"-->
-
-<section id="action-serialization-json">
-<h3><code>actions</code></h3>
-
-  <p>
-    In a <code>Thing</code> instance, the value assigned to <code>actions</code>
-    is a <a>Map</a> of instances of <code>ActionAffordance</code>.
-    <span class="rfc2119-assertion" id="td-actions">
-    All name-value pairs of a <a>Map</a> of <code>ActionAffordance</code> instances
-    MUST be serialized as members of the JSON object that results from serializing the <a>Map</a>;
-    the name of a pair MUST be serialized as a JSON string and the value of the pair, an
-    instance of <code>ActionAffordance</code>, MUST be serialized as a JSON object.
-  </span></p>
-
-  <p><span class="rfc2119-assertion" id="td-action-names">
-    All name-value pairs of an instance of <code>ActionAffordance</code>,
-    where the name is a <a>Vocabulary Term</a> included in (one of) the <a>Signatures</a> of
-    <code>ActionAffordance</code> or <code>InteractionAffordance</code>,
-    MUST be serialized as members of the JSON object that results from
-    serializing the <code>ActionAffordance</code> instance, with the
-    <a>Vocabulary Term</a> as name.
-  </span></p>
 
   <p>
-  <span class="rfc2119-assertion" id="td-action-objects">
-    The values assigned to <code>input</code> and <code>output</code> in an instance of
-    <code>ActionAffordance</code>
-    MUST be serialized as JSON objects.
+  The JSON members named <code>titles</code> and <code>descriptions</code> are 
+  used within the TD document to provide human-readable metadata
+  in multiple languages within a single TD document.
+  <span class="rfc2119-assertion" id="td-multi-languages">
+    All name-value pairs of a <code>MultiLanguage</code> <a>Map</a>
+    MUST be serialized as members of a JSON object,
+    where the name is a well-formed language tag as defined by [[!BCP47]]
+    and the value the human-readable string.
   </span>
-  They rely on the the <a>Class</a> <a href="#dataschema" class="sec-ref"><code>DataSchema</code></a>,
-  whose serialization is defined in <a href="#data-schema-serialization-json"></a>.
-  </p>
-
-  <p><span class="rfc2119-assertion" id="td-action-arrays">
-    The value assigned to <code>forms</code> in an instance of <code>ActionAffordance</code>
-    MUST be serialized as a JSON array
-    containing one or more JSON object serializations as defined in <a href="#form-serialization-json"></a>.
-  </span></p>
-
-  <p>
-  A TD snippet of an Action affordance is given below:
+  See <a href="#multilanguage"></a> for details.
   </p>
 
-<aside class="example" id="action-serialization-sample" title="Sample of an Action serialization">
-<pre>
+  <p>A TD snippet using <code>titles</code> and <code>descriptions</code> at different levels is given below:</p>
+    
+<pre class="example">
+{
+    "@context": "https://www.w3.org/2019/td/v1",
+    "title": "MyThing",
```

Comment:
I find it odd that this example has a title but no titles and descriptions but no description at the Thing level.   It's not wrong according to the spec, just odd.   On the other hand, such an example would probably break content negotiation.   I see that later this is expanded upon, but I kind of wonder at the value of this particular example.   Maybe should just skip it, and use a simple title+description one and another with paired title+titles and description+descriptions.

Reply:
Fixed.